website/docs/r/storage_account.html.markdown

```diff
@@ -249,6 +253,40 @@ any combination of `Logging`, `Metrics`, `AzureServices`, or `None`.
 
 ---
 
+A `azure_files_identity_based_authentication` block supports the following:
```

Comment:
I think this is unclear, is this to configure files authentication? so shouldn't it be
```suggestion
A `azure_files_authentication` block supports the following:
```